packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.constants.js

```diff
@@ -1,3 +1,13 @@
+const TRACKING_PREFIX =
+  'dedicated::dedicatedCloud::details::datacenter::details::dashboard::nsx::';
+export const ADD_HOST_TRACKING_PREFIX = `${TRACKING_PREFIX}::add-host`;
+export const ORDER_HOST_TRACKING_PREFIX = `${TRACKING_PREFIX}::add-storage`;
+export const CONFIRM_TRACKING_PREFIX = `${TRACKING_PREFIX}::confirm`;
+export const CANCEL_TRACKING_PREFIX = `${TRACKING_PREFIX}::cancel`;
+export const NSX_TRACKING_PREFIX = `${TRACKING_PREFIX}::nsx`;
+export const CONFIRM_SUCCESS_TRACKING_PREFIX = `${TRACKING_PREFIX}::nsx-success`;
+export const CONFIRM_ERROR_TRACKING_PREFIX = `${TRACKING_PREFIX}::nsx-error`;
```

Comment:
You should group your tracking constants in a global object
```
export const TRACKING = {
 ADD_HOST: `${TRACKING_PREFIX}::add-host`,
...
}
```
You can rename also those constants name because most of them are not a prefix value